Readme.md

```diff
@@ -1,6 +1,8 @@
 # Bref Extra PHP Extension
 
-This repository has some AWS Lambda layers with PHP extensions. This is useful when you want something "off the shelf".
+This repository provides PHP extensions for [Bref](https://bref.sh/) applications via AWS Lambda layers.
```

Comment:
I just figured it may be worth adding a link back to Bref, so that visitors that stumble upon this repository from Google have a way to get a bit more context.

(not related to the PR, sorry)